src/json-editor.component.ts

```diff
@@ -73,6 +73,7 @@ export class JsonEditorComponent extends AbstractTrackerComponent implements OnI
   previews: Array<Preview> = [];
   isPreviewerHidden: boolean;
   keys: Set<string>;
+  paths: {[field: string]: Array<string>} = {};
```

Comment:
it's small but still repeated several times so maybe it's better to create a interface for this type like `PathCache` in `./shared/interfaces`